website/docs/configure_hydra/workdir.md

```diff
@@ -55,3 +55,15 @@ hydra:
         item_sep: ','
         exclude_keys: []
 ```
+
+### Customizing outputs with substitution through the CLI 
+
+Outputs can also be configured through the CLI, like any other configuration.
+
+> python3 train.py model.nb_layers=3 hydra.run.dir=3_layers
+
+While this feature by itself is not surprising, it can become really powerful to write multiruns without boilerplate using substitution.
```

Comment:
You are a bit apologetic here, no need :)

Reply:
Changed to be more doc-style and less blog-style